src/Analyzer.TemplateProcessor/ArmTemplateProcessor.cs

```diff
@@ -382,9 +390,12 @@ private void ProcessTemplateResourceLanguageExpressions(TemplateResource templat
                         evaluationContext: evaluationHelper.EvaluationContext);
                 }
             }
-            catch
+            catch (Exception ex)
             {
                 // Do not throw if there was an issue with evaluating language expressions
+
+                logger?.LogError(ex, "An exception occurred while evaluating the properties of a resource. Properties: {properties}", templateResource?.Properties?.Value);
```

Comment:
minor: null-coalescing operator isn't needed here, because if `templateResource` or `templateResource.Properties` was null, nothing would happen in the `try` block.